src/csvToJson.js

```diff
@@ -12,20 +12,17 @@ import { joinPath } from './utils';
 // is there some intersections, etc.
 // I think we can improve it very easy.
 
-const maxEntries = 10000;
-let result = [];
+
 let folderName;
-let numberOfFiles;
 
-const generateJsonFiles = (i, fileName, start, stop) => {
-  const data = result.slice(start, stop);
+
+const generateJsonFiles = (i, fileName, start, stop, dataEntries) => {
+  const data = dataEntries.slice(start, stop);
 
   // @TODO change that
   // we can also create a method for path.join, so it wouldn't complicate our code
   // really bad line
-  const jsonFileName = `${folderName}/${fileName}${i}.json`;
```

Comment:
I don't think it should be removed. there a lot of differences at how our datasets located at this moment, so I assume this line is part of the old version of the parser. it better to comment it, but not delete while we don't perfected this module

Reply:
it's actually part of prev. line. so I'm not sure if this edit wouldn't cause any future bugs